sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/OrchestrationService.java

```diff
@@ -0,0 +1,268 @@
+package com.sap.ai.sdk.app;
```

Comment:
(Major/Question)

Can you move the class to a dedicated `services` package/namespace?
Or are there issues with testing?